Pods not created with 1.1.2

I am running into a problem following the tutorial here https://infinispan.org/infinispan-operator/1.1.x/operator.html 

When running operator 1.1.2 the command `kubectl apply -f cr_minimal.yaml` does not create any pods.

cr_minimal.yaml
``` yaml
apiVersion: infinispan.org/v1
kind: Infinispan
metadata:
  name: example-infinispan
spec:
  replicas: 2
```

I have tried this in minikube running kubernetes 1.18 and amazon eks running kubernetes 1.16.8. With client version `Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"darwin/amd64"}`


I installed the operator using the instructions here https://operatorhub.io/operator/infinispan/stable/infinispan-operator.v1.1.2

The custom operator seems to exist
``` bash
$ kubectl get csv -n my-infinispan
NAME                         DISPLAY      VERSION   REPLACES   PHASE
infinispan-operator.v1.1.2   Infinispan   1.1.2                Succeeded
```

The event stream seems to show that the operator is running. 
``` bash
my-infinispan    22m         Normal    RequirementsNotMet        clusterserviceversion/infinispan-operator.v1.1.2   one or more requirements couldn't be found
my-infinispan    22m         Normal    RequirementsUnknown       clusterserviceversion/infinispan-operator.v1.1.2   requirements not yet checked
my-infinispan    22m         Normal    Scheduled                 pod/infinispan-operator-85bf78576-gz9nk            Successfully assigned my-infinispan/infinispan-operator-85bf78576-gz9nk to minikube
my-infinispan    22m         Normal    InstallWaiting            clusterserviceversion/infinispan-operator.v1.1.2   installing: waiting for deployment infinispan-operator to become ready: Waiting for rollout to finish: 0 out of 1 new replicas have been updated...
my-infinispan    22m         Normal    InstallWaiting            clusterserviceversion/infinispan-operator.v1.1.2   installing: waiting for deployment infinispan-operator to become ready: Waiting for deployment spec update to be observed...
my-infinispan    22m         Normal    AllRequirementsMet        clusterserviceversion/infinispan-operator.v1.1.2   all requirements found, attempting install
my-infinispan    22m         Normal    ScalingReplicaSet         deployment/infinispan-operator                     Scaled up replica set infinispan-operator-85bf78576 to 1
my-infinispan    22m         Normal    SuccessfulCreate          replicaset/infinispan-operator-85bf78576           Created pod: infinispan-operator-85bf78576-gz9nk
my-infinispan    22m         Normal    InstallWaiting            clusterserviceversion/infinispan-operator.v1.1.2   installing: waiting for deployment infinispan-operator to become ready: Waiting for rollout to finish: 0 of 1 updated replicas are available...
my-infinispan    22m         Normal    InstallSucceeded          clusterserviceversion/infinispan-operator.v1.1.2   waiting for install components to report healthy
my-infinispan    22m         Normal    Pulling                   pod/infinispan-operator-85bf78576-gz9nk            Pulling image "quay.io/infinispan/operator:1.1.2.Final"
my-infinispan    21m         Normal    Pulled                    pod/infinispan-operator-85bf78576-gz9nk            Successfully pulled image "quay.io/infinispan/operator:1.1.2.Final"
my-infinispan    21m         Normal    Created                   pod/infinispan-operator-85bf78576-gz9nk            Created container infinispan-operator
my-infinispan    21m         Normal    InstallSucceeded          clusterserviceversion/infinispan-operator.v1.1.2   install strategy completed with no errors
my-infinispan    21m         Normal    Started                   pod/infinispan-operator-85bf78576-gz9nk            Started container infinispan-operator
```

Getting the namespace returns what look like appropriate values
``` bash
 kubectl get all -n my-infinispan
NAME                                      READY   STATUS    RESTARTS   AGE
pod/infinispan-operator-85bf78576-gz9nk   1/1     Running   0          40m

NAME                              TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
service/example-infinispan-site   LoadBalancer   10.105.58.186   <pending>     7900:30578/TCP   18m

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/infinispan-operator   1/1     1            1           40m

NAME                                            DESIRED   CURRENT   READY   AGE
replicaset.apps/infinispan-operator-85bf78576   1         1         1       40m

```


Running the apply looks like it works
``` bash
kubectl apply -f cr_minimal.yaml
infinispan.infinispan.org/example-infinispan created
```
but no new pods appear and no events occur on the event stream.

All of the outputs are from minkube running on a mac, but the outputs from the eks cluster are nearly identical.

Please let me know if there is any additional information I can provide. I am hoping I just missed a step somewhere.

I knew I was just missing something. That did it, thank you! This was my first foray into operators so I wasn't positive how all the name spacing worked. I might recommend adding a namespace flag for the tutorial in section 2.2 https://infinispan.org/infinispan-operator/1.1.x/operator.html, though you may have no control over that.

 Now I am seeing one pod start to spin up. It is immediately crashing, but this may not be the platform for that stack trace. 

```
16:15:34,276 FATAL (main) [org.infinispan.SERVER] ISPN080028: Infinispan Server failed to start java.util.concurrent.ExecutionException: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheConfigurationException: ISPN000512: Cannot acquire lock '/opt/infinispan/server/data/___global.lck' for persistent global state
        at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
        at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
        at org.infinispan.server.Bootstrap.runInternal(Bootstrap.java:139)
        at org.infinispan.server.tool.Main.run(Main.java:98)
        at org.infinispan.server.Bootstrap.main(Bootstrap.java:39)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.infinispan.server.loader.Loader.main(Loader.java:71)
Caused by: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheConfigurationException: ISPN000512: Cannot acquire lock '/opt/infinispan/server/data/___global.lck' for persistent global state
        at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:760)
        at org.infinispan.manager.DefaultCacheManager.start(DefaultCacheManager.java:726)
        at org.infinispan.server.SecurityActions.lambda$startCacheManager$1(SecurityActions.java:51)
        at org.infinispan.security.Security.doPrivileged(Security.java:46)
        at org.infinispan.server.SecurityActions.doPrivileged(SecurityActions.java:31)
        at org.infinispan.server.SecurityActions.startCacheManager(SecurityActions.java:54)
        at org.infinispan.server.Server.run(Server.java:286)
        ... 8 more
Caused by: org.infinispan.commons.CacheConfigurationException: ISPN000512: Cannot acquire lock '/opt/infinispan/server/data/___global.lck' for persistent global state
        at org.infinispan.globalstate.impl.GlobalStateManagerImpl.acquireGlobalLock(GlobalStateManagerImpl.java:87)
        at org.infinispan.globalstate.impl.GlobalStateManagerImpl.start(GlobalStateManagerImpl.java:64)
        at org.infinispan.globalstate.impl.CorePackageImpl$1.start(CorePackageImpl.java:34)
        at org.infinispan.globalstate.impl.CorePackageImpl$1.start(CorePackageImpl.java:27)
        at org.infinispan.factories.impl.BasicComponentRegistryImpl.invokeStart(BasicComponentRegistryImpl.java:587)
        at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:578)
        at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
        at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
        at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
        at org.infinispan.factories.AbstractComponentRegistry.internalStart(AbstractComponentRegistry.java:341)
        at org.infinispan.factories.AbstractComponentRegistry.start(AbstractComponentRegistry.java:237)
        at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:755)
        ... 14 more
Caused by: java.io.FileNotFoundException: /opt/infinispan/server/data/___global.lck (Permission denied)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:187)
        at org.infinispan.globalstate.impl.GlobalStateManagerImpl.acquireGlobalLock(GlobalStateManagerImpl.java:81)
```

--- a/deploy/operator.yaml
+++ b/deploy/operator.yaml
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: infinispan-operator
       containers:
         - name: infinispan-operator
-          image: jboss/infinispan-operator:latest
+          image: jboss/infinispan-operator:1.1.2
           ports:
           - containerPort: 60000
             name: metrics

